cloud/linode/cloud.go

```diff
@@ -98,7 +100,15 @@ func newCloud() (cloudprovider.Interface, error) {
 	url := os.Getenv(urlEnv)
 	ua := fmt.Sprintf("linode-cloud-controller-manager %s", linodego.DefaultUserAgent)
 
-	linodeClient, err := client.New(apiToken, ua, url)
+	// set timeout used by linodeclient for API calls
+	timeout := client.DefaultClientTimeout
+	if raw, ok := os.LookupEnv("LINODE_REQUEST_TIMEOUT"); ok {
```

Comment:
We want to rename this var to indicate the time unit?

Reply:
Updated.